wondrous-app/components/Settings/Payouts/styles.tsx

```diff
@@ -24,23 +281,243 @@ export const TableCellText = styled(Typography)`
 
 export const StyledCheckbox = styled(Checkbox)`
   && {
+    width: 20px;
+    height: 20px;
+    background: ${(props) => (props.checked ? palette.white : 'transparent')} !important;
+    color: ${(props) => (props.disabled ? palette.grey800 : palette.highlightPurple)} !important;
+    opacity: ${(props) => (props.disabled ? 0.5 : 1)};
+  }
+`;
+
+export const PayeeProfileLink = styled.a`
+  text-decoration: none;
+`;
+
+export const PayeePayButton = styled(StyledButton)`
```

Comment:
use Button from `components/Button`, if button has new style, add styles to the theme there and update storybook